options/locale/locale_en-US.ini

```
@@ -170,7 +170,7 @@ string.desc = Z - A
 
 [error]
 occurred = An error occurred
-report_message = If you are sure this is a Gitea bug, please search for issues on <a href="https://github.com/go-gitea/gitea/issues" target="_blank">GitHub</a> or open a new issue if necessary.
+report_message = If believe that this is a Gitea bug, please search for issues on <a href="https://github.com/go-gitea/gitea/issues" target="_blank">GitHub</a> or open a new issue if necessary.
```

```suggestion
report_message = If you believe that this is a Gitea bug, please search for issues on <a href="https://github.com/go-gitea/gitea/issues" target="_blank">GitHub</a> or open a new issue if necessary.
```

---

Oh, now is also a good time to get rid of the hardcoded link:

```suggestion
report_message = If you believe that this is a Gitea bug, please search for issues on <a href="%[1]s" target="_blank">GitHub</a> or open a new issue if necessary.
```